examples/tests/sdk_tests/schedule/init.test.w

```diff
@@ -5,6 +5,8 @@ skip: true
 
 bring cloud;
 
+new cloud.Schedule( rate: 5m ) as "s0";
+
```

Comment:
What does this test?

Reply:
In this file we are testing the schedule constructor, (since we have a few ways to init it, and we want to avoid regression)- 
I've added a valid one on the top of the file since it is complaining about an empty configuration file if not added (it does the same with the awscdk target)

Reply:
👍